flink-shaded-netty-4/src/main/resources/META-INF/NOTICE

```diff
@@ -6,4 +6,36 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
-- io.netty:netty-all:4.1.65.Final
```

Comment:
netty-all is no longer a fat-jar and instead pulls in everything as a transitive dependency.